shared/css/conversation.css

```diff
@@ -376,6 +376,34 @@ html[dir="rtl"] .settings-menu.dropdown-menu {
   }
 }
 
+/* Stream paused */
+.focus-stream > .room-inner-info-area > .remote-stream-paused {
+  background-color: #fff;
+  border: solid 1px #a6a6a6;
+  box-shadow: 0 2px 4px rgba(0,0,0,.25);
+  display: inline-block;
+  margin: auto;
+  padding: 10px 15px;
+}
+
+.focus-stream > .room-inner-info-area > .remote-stream-paused > h1 {
+  font-size: 1.6rem;
+  line-height: 3rem;
+  padding-left: 42px;
+  position: relative;
+  text-align: left;
```

Comment:
These classes will need rtl versions of them specifying.

Please also add a paused view to the ui-showcase so that we can easily view it and inspect for differences.